iosApp/KotlinConf.xcodeproj/project.pbxproj

```diff
@@ -348,7 +348,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 3.2.2;
+				MARKETING_VERSION = 25.0.0;
```

Comment:
this change will trigger long full app review after submission. I would avoid this for minor updates

Reply:
Don't we want to have one full app review for the 2025 version anyway?

Do you think we should stick with 3.x.x for the entire 2025 year instead? (In that case, we'd have to change the Android version to that as well.)

Reply:
we usually have a short one for the patch updates